### ISSUE TITLE
Display all media attachments of a toot

### ISSUE THREAD

**Issue by hdurer** [martianhi/mastodon.el#110]
Currently `mastodon-tl--media` adds only the first (if any) media preview to the buffer.

**Comment by alexjgriffith** [comment]
Sample with multiple attachments

``` elisp
((id . 4833540)
 (created_at . "2017-05-01T17:44:31.132Z")
 (in_reply_to_id)
 (in_reply_to_account_id)
 (sensitive)
 (spoiler_text . "")
 (visibility . "public")
 (application
  (name . "mastodon.el")
  (website . "https://github.com/jdenen/mastodon.el"))
 (account
  (id . 34105)
  (username . "h_d")
  (acct . "h_d")
  (display_name . "Holger Durer")
  (locked . :json-false)
  (created_at . "2017-04-03T13:18:32.233Z")
  (followers_count . 19)
  (following_count . 27)
  (statuses_count . 23)
  (note . "<p>Ageing programmer. Likes Lisp, Clojure, LFE, Erlang, Elm, \342\200\246 but does little with them.<br />Pronoun: he<br />Toot at me in English, Deutsch, Esperanto</p>")
  (url . "https://mastodon.social/@h_d")
  (avatar . "https://files.mastodon.social/accounts/avatars/000/034/105/original/3da54479ac3094bf.jpg")
  (avatar_static . "https://files.mastodon.social/accounts/avatars/000/034/105/original/3da54479ac3094bf.jpg")
  (header . "https://files.mastodon.social/accounts/headers/000/034/105/original/a09348915de8a74b.jpg")
  (header_static . "https://files.mastodon.social/accounts/headers/000/034/105/original/a09348915de8a74b.jpg"))
 (media_attachments .
		    [])
 (mentions .
	   [])
 (tags .
       [])
 (uri . "tag:mastodon.social,2017-05-01:objectId=4833540:objectType=Status")
 (content . "<p>From reading the source code of mastodon.el I would assume that this will display multiple images inline of this toot:</p><p>Rick no1:<br />Media_Link:: <a href=\"https://media.giphy.com/media/kFgzrTt798d2w/giphy.gif\" rel=\"nofollow noopener\" target=\"_blank\"><span class=\"invisible\">https://</span><span class=\"ellipsis\">media.giphy.com/media/kFgzrTt7</span><span class=\"invisible\">98d2w/giphy.gif</span></a></p><p>Rick no2:<br />Media_Link:: <a href=\"http://gph.is/1JeERZf\" rel=\"nofollow noopener\" target=\"_blank\"><span class=\"invisible\">http://</span><span class=\"\">gph.is/1JeERZf</span><span class=\"invisible\"></span></a></p><p>Kitten no1: \360\237\220\261<br />Media_Link:: <a href=\"http://cdn3-www.cattime.com/assets/uploads/2011/08/best-kitten-names-1.jpg\" rel=\"nofollow noopener\" target=\"_blank\"><span class=\"invisible\">http://</span><span class=\"ellipsis\">cdn3-www.cattime.com/assets/up</span><span class=\"invisible\">loads/2011/08/best-kitten-names-1.jpg</span></a></p><p>Kitten no2: <br />Media_Link:: <a href=\"http://dirtypawzpetgrooming.com/wp-content/uploads/2012/01/kitten-little.jpg\" rel=\"nofollow noopener\" target=\"_blank\"><span class=\"invisible\">http://</span><span class=\"ellipsis\">dirtypawzpetgrooming.com/wp-co</span><span class=\"invisible\">ntent/uploads/2012/01/kitten-little.jpg</span></a></p><p>\342\230\272</p>")
 (url . "https://mastodon.social/@h_d/4833540")
 (reblogs_count . 0)
 (favourites_count . 0)
 (reblog)
 (favourited)
 (reblogged))

```

**Comment by alexjgriffith** [comment]
Closed by #111
